src/devtools/client/inspector/rules/models/element-style.ts

```diff
@@ -406,11 +406,9 @@ export default class ElementStyle {
 
       // Collect all relevant CSS declarations (aka TextProperty instances).
       if (filterCondition) {
-        for (const textProp of rule.textProps.slice(0).reverse()) {
-          if (textProp.enabled) {
-            textProps.push(textProp);
-          }
-        }
+        const enabledTextProps = rule.textProps.filter(textProp => textProp.enabled);
+        enabledTextProps.reverse();
+        textProps.push(...enabledTextProps);
```

Comment:
Especially since this PR is about optimizing things– this new bit looks like it could be optimized too, to avoid creating a throw-away array and having to `.reverse` it and spread it.

```js
for (let index = rule.textProps.length - 1; index >= 0; index--) {
  if (textProp.enabled) {
    textProps.push(textProp);
  }
}
```